test/standalone/main.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable id-length */
+// @flow
+
+const webpack = require('webpack');
+const requireFromString = require('require-from-string');
+const MemoryFS = require('memory-fs');
+const path = require('path');
+const config = require('./webpack.config');
+
+const fs = new MemoryFS();
+const compiler = webpack(config);
+
+// $ExpectError
+compiler.outputFileSystem = fs;
```

Comment:
can you explain this one to me? I am not a webpack ninja

Reply:
I've never managed to get Flow happy about specifiing a outputFileSystem, but i dont like to cleanup everything webpack outputs